scripts/generate_test_certs.sh

```diff
@@ -21,6 +21,11 @@ ${CERTSTRAP} --depot-path "${depot_path}" init --passphrase '' --common-name log
 mv -f "${depot_path}"/loggregatorCA.crt "${depot_path}"/loggregator-ca.crt
 mv -f "${depot_path}"/loggregatorCA.key "${depot_path}"/loggregator-ca.key
 
+# CA to distribute to dummy gorouter ca certs
+${CERTSTRAP} --depot-path "${depot_path}" init --passphrase '' --common-name gorouterCA --years "20"
```

Comment:
Not related to this change

Certrap lib is not maintained since atleast 2 years (considering the last commit). Look's like , it is silently unmaintained. Should we think an alternative ?

Reply:
Created issue #3544